File: coding_practice/Q1/hello.py
def get_name_from_user():
    name = raw_input("What's your name ?")
    return name

def make_text( name ) :
    return "Hello, " + name + ", nice to meet you"

def print_text( text ) :
    print text 

if __name__ ==  "__main__" :
    print_text( make_text( get_name_from_user()))

File: coding_practice_goversion/Q1/hello.go
package main

import (
	"fmt"
)

func getNameByUser() string{
        var str string
        fmt.Printf("What's your name? ")
	fmt.Scanf("%s", &str)
        return str
}

func makeText( text string ) string{
        return "Hello, " + text + ", nice to meet you" 
}

func printText( text string ){
        fmt.Println(text)
}
func main() {
	fmt.Println("You will see Hello World message with function called")
        printText(makeText(getNameByUser()))
}
